Dockerfile

```diff
@@ -135,7 +135,7 @@ RUN CHART_VERSION="v3.1906"                   CHART_FILE=/charts/rke2-calico.yam
 RUN CHART_VERSION="v1.0.006"                  CHART_FILE=/charts/rke2-calico-crd.yaml     CHART_BOOTSTRAP=true   /charts/build-chart.sh
 RUN CHART_VERSION="1.10.101-build2021022304"  CHART_FILE=/charts/rke2-coredns.yaml        CHART_BOOTSTRAP=true   /charts/build-chart.sh
 RUN CHART_VERSION="3.30.003"                  CHART_FILE=/charts/rke2-ingress-nginx.yaml  CHART_BOOTSTRAP=false  /charts/build-chart.sh
-RUN CHART_VERSION="v1.21.1-build2021052004"   CHART_FILE=/charts/rke2-kube-proxy.yaml     CHART_BOOTSTRAP=true   /charts/build-chart.sh
+RUN CHART_VERSION="v1.21.2-build2021061701"   CHART_FILE=/charts/rke2-kube-proxy.yaml     CHART_BOOTSTRAP=true   /charts/build-chart.sh
```

Comment:
This won't work because the assets have been moved as a side-effect of refactoring into separate `rke2-kube-proxy-1.##` packages upstream

Reply:
I think you may have commented on an out of date version. That's what I have in that file.